src/celeritas/ext/detail/GeantPhysicsList.cc

```diff
@@ -296,7 +308,8 @@ void GeantPhysicsList::add_e_processes(G4ParticleDefinition* p)
         auto process = std::make_unique<G4eMultipleScattering>();
 
         if (options_.msc == MscModelSelection::urban
-            || options_.msc == MscModelSelection::all)
+            || options_.msc == MscModelSelection::urban
```

Comment:
Duplicate `MscModelSelection::urban` condition.

Reply:
Meant for this to be `urban_extended`.